### PR TITLE
Edit media messages

### PR DESCRIPTION
**Pull request by Arseny271**

### CODE REVIEW

**Review by MonirXoxo — COMMENTED** [review]
Ok

File: app/src/main/java/org/thunderdog/challegram/data/ContentPreview.java
Comment (vkryl) [inline app/src/main/java/org/thunderdog/challegram/data/ContentPreview.java:1]: The idea of this class is to be non-modifiable. If the new preview is needed, then the new `ContentPreview` instance could be created.

File: app/src/main/java/org/thunderdog/challegram/data/ContentPreview.java [hunk 1]
```diff
@@ -86,8 +86,8 @@ public class ContentPreview {
   private static final int ARG_RECURRING_PAYMENT = -3;
   private static final long ADDITIONAL_MESSAGE_UI_LOAD_TIMEOUT_MS = -1; // Always async
   private static final long ADDITIONAL_MESSAGE_LOAD_TIMEOUT_MS = 0;
-  public final @Nullable Emoji emoji, parentEmoji;
-  public final @StringRes int placeholderText;
+  public @Nullable Emoji emoji, parentEmoji;
+  public @StringRes int placeholderText;
```
Comment (vkryl) [inline app/src/main/java/org/thunderdog/challegram/data/ContentPreview.java:90]:
```suggestion
  public final @Nullable Emoji emoji, parentEmoji;
  public final @StringRes int placeholderText;
```

**Review by renuac85 — COMMENTED** [review]
Innitials Commits and Merges

**Review by Qtalggrm — COMMENTED** [review]
07718596084

**Review by Yoggie9477 — COMMENTED** [review]
- [ ]

**Review by Vfu211 — COMMENTED** [review]
Hdriihsiriidherji-$((&+84(3(6hhrptohwoejiedkjrtt

**Comment by Vfu211** [comment]
Hsiif8ejuwhfitouu;($$(&;"+$(+=¥§{¥{¢§€{÷•§

**Comment by Qtalggrm** [comment]
*Yoggie9477*

في الأحد، ٣١ مارس ٢٠٢٤, ٩:١٥ ص Yoggie9477 ***@***.***> كتب:

**Pull request merged**